about/bots.md

```diff
@@ -38,9 +38,9 @@ You can find Dotsimus' website [here](https://dotsimus.com/). Additionally, you
 | Flareon             | [https://github.com/Ignaciox](https://github.com/Ignaciox/)                                           | Music                   |
 | Heartosoft          | [https://github.com/Microsoft-community/starboard](https://github.com/Microsoft-community/starboard)  | Starboard               |
 | iReport             | [https://discord.gg/apple](https://discord.gg/apple/)                                                 | Inter-server reporting  |
-| Microsoft Community | [http://mee6.xyz](http://mee6.xyz/)                                                                   | Levelling and utilities |
+| Microsoft Community | [http://mee6.xyz](http://mee6.xyz/)                                                                   | Leveling and utilities  |
```

Comment:
Levelling is valid in British English, its not a typo

Reply:
Maybe that's what being used to American English does to me lol. But I think some articles here are inconsistent in this area (usage of British vs American) so I don't really know if I should keep or revert this.

Reply:
I would revert it for now pending a proper discussion about which English we want to use for the site, if we decide it necessary to be consistent

Reply:
Alright, done.